src/csharp/build_packages_dotnetcli.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
this file has been deleted in upstream/master. Let's delete from this PR as well.

Reply:
FTR https://github.com/grpc/grpc/pull/16208/files#diff-f8017b26a71fa84b0b616110830f9858